pluto.jl

```diff
@@ -126,26 +126,40 @@ $$[CPU] ↔ [RAM] ↔ [硬盘缓存] ↔ [硬盘]$$
 
 # ╔═╡ f58d428c-8aef-11eb-3127-89d729e23823
 md"""
-## Avoid cache misses
-The RAM is faster than the disk, and the CPU in turn is faster than RAM. A CPU ticks like a clock, with a speed of about 3 GHz, i.e. 3 billion ticks per second. One "tick" of this clock is called a *clock cycle*. While this is a simplification, you may imagine that every cycle, the CPU executes a single, simple command called a *CPU instruction* which does one operation on a small piece of data. The clock speed then can serve as a reference for other timings in a computer. It is worth realizing just how quick a clock cycle is: In one cycle, a photon will travel only around 10 cm. In fact, modern CPUs are so fast that a significant constraint on their physical layout is that one must take into account the time needed for electricity to move through the wires inside them, so called wire delays.
+## 避免缓存未命中
+RAM 比内存快，而 CPU 又比 RAM 快。CPU滴答类似于时钟，速度约为 3 GHz，即每秒滴答30亿次。
+CPU 时钟的一次**滴答**被称为**时钟周期**。
+虽然这做了简化，但是你可以想象，在每个周期内，CPU都会执行一条简单的**指令**，该指令执行一次对小块数据的操作。
+时钟速度可以作为计算机中其他计时器的参考。
+了解时钟周期到底有多快是很值得的：光在一个时钟周期内只前进 10 cm。
+事实上，现代 CPU 如此之快，以至于一个约束物理布局的重要因素是必须考虑电子在内部线路传输所花的时间，这被称为线路延迟。
```

Comment:
在出现从句的时候，一般可以把从句摘出来单独成句翻译。只要保证连词的逻辑被正确翻译即可。

```suggestion
在设计现代 CPU 时一个重要的限制因素就是线路延迟，即电子在 CPU 内部线路传输所花的时间。这恰恰是因为 CPU 运算的太快了。
```

Reply:
嗯。